widgetsnbextension/src/embed_widgets.js

```diff
@@ -11,7 +11,7 @@ var embed_widgets = function() {
                 'drop_defaults': true
             }).then(function(state) {
                 var data = JSON.stringify(state, null, '    ');
-                var value = '<script src="http://perso.crans.org/corlay/built/index.built.js"></script><script>w(' + data + ');</script>';
+                var value = '<script src="https://npmcdn.com/jupyter-js-widgets@0.0.17/dist/embed.js"></script><script>w(' + data + ');</script>';
```

Comment:
If we can get the version of the package, that is even better. Otherwise this will need to be added to the release instructions.